server.go

```diff
@@ -75,7 +74,7 @@ type Server struct {
 
 	startTime time.Time
 
-	OpenAPIConfig OpenAPIConfig
+	OpenAPIServerConfig
```

Comment:
Not a fan of embedding if it's not to inherit methods (in order to satisfy an interface) or for backward-compatibility reasons.

By doing useless embedding, users will be able to access via `s.<field>` instead of `s.OpenAPIServerConfig.<field>` and I think it's not a good thing if ther isn't a good reason to do so (conciseness isn't a good one IMHO).

Reply:
Done